src/main/java/games/strategy/engine/lobby/server/userDB/DerbyUserController.java

```diff
@@ -156,18 +165,20 @@ public boolean login(final String userName, final String hashedPassword) {
   }
 
   @Override
-  public DbUser getUser(final String userName) {
+  public DbUser getUserByName(final String userName) {
```

Comment:
In part this is a reflection of the existing names. To another extent, it is 'user name' and called that pretty consistently. In honesty the 'user' prefix on everything is a potential concern and likely not very helpful and probably should be removed.
With that said, I'm not terribly bothered by the current variables. We could rename this method to `getUserByUserName`, but that seems really wordy to me.